### PR TITLE
Updating metatranscriptome documentation

### PR DESCRIPTION
**Pull request by smruthi98**
Adding metatranscriptome processing documentation: Sortmerna for ribosomal read filtering

### CODE REVIEW

File: qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst [hunk 1]
```diff
@@ -152,3 +152,54 @@ Shogun reference databases
...
+Ribosomal read filtering
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+`SortMeRNA <https://bioinfo.lifl.fr/RNA/sortmerna/>`_
+is used for removal of ribosomal reads from quality filtered metaT data
+
+Latest SortMeRNA version: v2.1
+
+Input: Quality filtered metaT reads (Fasta/Fastq) 
+Ribosomal reads are identified by searching against pre-curated rRNA databases. Currently, rRNA databases covering bacteria, archaea and eukarya were downloaded from the SILVA and rfam databases and indexed.
```
Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst:173]:
```suggestion
Ribosomal reads are identified by searching against pre-curated rRNA databases. Currently, rRNA databases covering bacteria, archaea and eukarya were downloaded and indexed from `SILVA <https://www.arb-silva.de>`_ and `Rfam <https://rfam.xfam.org>`_.
```

File: qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst [hunk 2]
```diff
@@ -152,3 +152,54 @@ Shogun reference databases
...
+Currently indexed databases and their clustering ids:
```
Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst:174]: If I am understanding correctly, the following is a list of the databases and the clustering thresholds that will be available on Qiita?

Comment (smruthi98) [reply qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst:174]: That's correct. Currently I have them downloaded, indexed and tested the databases in Barnacle.

**Comment by ElDeveloper** [comment]
Thanks @smruthi98!

**Pull request merged**